GVRf/Framework/framework/src/main/jni/vulkan/vulkanCore.cpp

```diff
@@ -955,6 +956,11 @@ void VulkanCore::InitPipelineForRenderData(const GVR_VK_Vertices* m_vertices, Vu
                 VulkanShader *shader = static_cast<VulkanShader *>(shader_manager->getShader(
                         rdata->get_shader(false,curr_pass)));
 
+                float line_width;
+                if(!rdata->pass(curr_pass)->material()->getFloat("line_width", line_width))
```

Comment:
material() could theoretically return nullptr, right?

Reply:
Theoretically yes, adding the check.
About RenderPass hashcode. In vulkan we are caching vulkan pipeline on that basis.
So if two Objects have different cull_types or shaderID should have different pipelines.